README.md

```diff
@@ -214,6 +214,94 @@ oc apply -f deploy/olm-catalog/subscription.yaml
 ```
 
 ### Creating a Windows MachineSet
+
+#### Pre-configuration steps in case of vSphere cluster:
```

Comment:
I dont think this should be written out here. I think it should be broken out into a seperate doc linked to under the https://github.com/openshift/windows-machine-config-operator/blob/master/README.md#pre-requisites section